### PR TITLE
Allowing any 2xx response from Cloud Feeds to count as a success

### PR DESCRIPTION
**Pull request by dmnjohns**

### CODE REVIEW

**Comment by dmnjohns** [comment]
test this please

**Comment by dkowis** [comment]
ok to test

File: cf-sink/src/main/scala/org/openrepose/flume/sinks/CloudFeedPublisher.scala [hunk 1]
```diff
@@ -28,10 +28,10 @@ class CloudFeedPublisher(feedsEndpoint: String, httpProperties: Map[String, Stri
     try {
       val statusCode = httpResponse.getStatusLine.getStatusCode
       statusCode match {
-        case HttpStatus.SC_OK =>
-          logger.debug("Successfully published to cloud feeds")
         case HttpStatus.SC_UNAUTHORIZED =>
           throw new UnauthorizedException("Feeds rejected the post as unauthorized")
+        case _ if statusCode / 100 == 2 =>
+          logger.debug("Successfully published to cloud feeds")
```
Comment (usnavi) [inline cf-sink/src/main/scala/org/openrepose/flume/sinks/CloudFeedPublisher.scala]: wha?!?  :-P

A successful post returns 201.  Am i missing something?  Just sticking my nose in yo bidness.

Comment (dkowis) [inline cf-sink/src/main/scala/org/openrepose/flume/sinks/CloudFeedPublisher.scala]: Apparently it doesn't? @dmnjohns brought it up

Comment (adrianjgeorge) [inline cf-sink/src/main/scala/org/openrepose/flume/sinks/CloudFeedPublisher.scala]: Please no integer math, it's non obvious what it does

Comment (adrianjgeorge) [inline cf-sink/src/main/scala/org/openrepose/flume/sinks/CloudFeedPublisher.scala]: OK is 200 not 201, so to solve this problem and future proof we are saying any 2xx is a success

**Pull request merged**